src/makeTxn.ts

```diff
@@ -64,53 +62,6 @@ export function makePaymentTxnWithSuggestedParams(
   return new txnBuilder.Transaction(o);
 }
 
-/**
- * makePaymentTxn takes payment arguments and returns a Transaction object
- * @param from - string representation of Algorand address of sender
- * @param to - string representation of Algorand address of recipient
- * @param fee - integer fee per byte, in microAlgos. for a flat fee, overwrite the fee property on the returned object
- * If the final calculated fee is lower than the protocol minimum fee, the fee will be increased to match the minimum.
- * @param amount - integer amount to send, in microAlgos
- * @param closeRemainderTo - optionally close out remaining account balance to this account, represented as string rep of Algorand address
- * @param firstRound - integer first protocol round on which this txn is valid
- * @param lastRound - integer last protocol round on which this txn is valid
- * @param note - uint8array of arbitrary data for sender to store
- * @param genesisHash - string specifies hash genesis block of network in use
- * @param genesisID - string specifies genesis ID of network in use
- * @param rekeyTo - rekeyTo address, optional
- * @Deprecated in version 2.0 this will change to use the "WithSuggestedParams" signature.
```

Comment:
@jasonpaulos Do we want to remove this function entirely? Or just change the signature so that it takes `WithSuggestedParams` as an argument?

Reply:
Treat with a grain of salt because my JS knowledge is limited - From my read, there's a direct replacement via `makePaymentTxnWithSuggestedParams`.

So it _seems_ to me that the PR is accurate.

Reply:
The changes here look to be in line with the discussion we had today. I.e., removing the deprecated `makeX` functions in favor of the newer `makeXWithSuggestedParams`.

Reply:
correct, I made the changes reflected in changelog for `make*Txn` in favor of `make*WithSuggestedParams`.